src/test/regress/expected/sequences.out

```diff
@@ -0,0 +1,16 @@
+SET search_path TO sequences_schema;
+-- see the sequence object
+select count(*) from pg_sequence where seqrelid = 'seq_0'::regclass;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+INSERT INTO seq_test_0 VALUES (1,2);
+-- verify that sequence works properly
+select max(z)<nextval('seq_0') as check_sanity from seq_test_0 ;
+ check_sanity
+---------------------------------------------------------------------
+ t
+(1 row)
```

Comment:
maybe few more creative tests like:

BEGIN;
INSERT INTO seq_test_0 VALUES (199999,9, DEFAULT);
SELECT setval('seq_0', max(x)) FROM seq_test_0;
SELECT currval('seq_0') = max(x) FROM seq_test_0;
COMMIT;

I find PG regression tests useful to find such interesting queries/commands/patterns: https://github.com/postgres/postgres/blob/master/src/test/regress/sql/sequence.sql

Like, rename a sequence on the create file, or use different syntax etc.
```SQL
-- test CYCLE and NO CYCLE
ALTER SEQUENCE sequence_test2 RESTART WITH 24
  INCREMENT BY 4 MAXVALUE 36 MINVALUE 5 CYCLE;
```

Reply:
This causes flakiness, arbitrarily erroring out:
ERROR:  setval: value 3 is out of bounds for sequence "renamed_seq" (281474976710657..562949953421313)